string tactic
int32 smashStrategy
---
bool success